lms/product/factory.py

```diff
@@ -35,7 +35,7 @@ def _get_family(
         return Product.Family.BLACKBOARD
 
     # If we are in an API request, where we are forwarding the product type ourselves
-    if request.content_type == "application/json":
+    if request.content_type == "application/json" and "lms" in request.json:
         return Product.Family(request.json["lms"]["product"])
 
     # In an LTI launch we'll use the parameters available to guess
```

Comment:
I can't comment on that line, but below in line 50 we mention `LTIOutcomesViews` that's a view that uses json but doesn't include the product hint.